app/models/user.rb

```diff
@@ -38,6 +38,18 @@ def external_user?
     is_a?(ExternalUser)
   end
 
+  def learner?
+    role == 0
```

Comment:
rubocop proposes `role.zero?` instead of `role == 0`
But I think we should use the same style for all three functions, right?